package.json

```diff
@@ -4,9 +4,7 @@
   "description": "",
   "main": ".eslintrc.js",
   "scripts": {
-    "lint": "./node_modules/.bin/eslint --format codeframe .",
-    "lint:fix": "./node_modules/.bin/eslint --fix --format codeframe .",
-    "test": "./node_modules/.bin/mocha tests.js"
+    "test": ".mocha ./test*.js"
```

Comment:
not sure why you took out your scripts it caused your tests not to even run. So you have no passing tests.